src/clincoded/static/components/variant_central/interpretation/segregation/evidenceSheet.js

```diff
@@ -166,7 +166,9 @@ let EvidenceSheet = createReactClass({
      * @param {array} inputIds 
      */
     checkForDuplicateHpo(inputIds) {
-        return inputIds.some((id, index) => inputIds.indexOf(id) !== index)
+        if (inputIds && inputIds.length) {
+            return inputIds.some((id, index) => inputIds.indexOf(id) !== index)
```

Comment:
```suggestion
            return inputIds.some((id, index) => inputIds.indexOf(id) !== index);
```
Didn't notice before but you forgot a semi-colon

Reply:
Got it, thanks for the quick turnaround with the review!